secure-ios-app/accesscontrol/AccessControlBuilder.swift

```diff
@@ -18,11 +19,20 @@ class AccessControlBuilder {
     
     func build() -> AccessControlRouter {
         let mainStoryboard = UIStoryboard(name: "Main", bundle: nil)
-        let authService = self.appComponents.resolveAuthService()
         let viewController = mainStoryboard.instantiateViewController(withIdentifier: "AccessControlViewController") as! AccessControlViewController
-        viewController.userIdentity = authService.currentIdentity()
+        viewController.userIdentity = self.resolveCurrentUser()
         
         let accessControlRouter = AccessControlRouterImpl(viewController: viewController)
         return accessControlRouter
     }
+    
+    func resolveCurrentUser() -> User? {
```

Comment:
@twnolan Can you also add a docs tag to this function. `resolveCurrentUser` should be ok for naming.